From 88605065049470b2617d466b23e42c06ead3733b Mon Sep 17 00:00:00 2001
From: Duncan Macleod <macleoddm@cardiff.ac.uk>
Date: Tue, 30 Jul 2024 16:18:57 +0100
Subject: [PATCH] update numpy default numeric types

closes #4
---
 pygsl_lite/__init__.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/pygsl_lite/__init__.py b/pygsl_lite/__init__.py
index 79927b2..ab40c9f 100644
--- a/pygsl_lite/__init__.py
+++ b/pygsl_lite/__init__.py
@@ -98,20 +98,20 @@ else:
 try:
     Int =  _numobj.Int
 except  AttributeError:
-    # For numpy 1.0
-    Int = _numobj.int_
+    # For numpy
+    Int = _numobj.int64
 
 try:
     Float = _numobj.Float
 except  AttributeError:
-    # For numpy 1.0
-    Float = _numobj.float_
+    # For numpy
+    Float = _numobj.float64
 
 try:
     Complex = _numobj.Complex
 except  AttributeError:
-    # For numpy 1.0
-    Complex = _numobj.complex_
+    # For numpy
+    Complex = _numobj.complex128
 
 try:
     ArrayType = _numobj.ArrayType
-- 
2.41.0

